packages/stylelint-plugin/test/hexColor.test.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2020 Palantir Technologies, Inc. All rights reserved.
```

Comment:
minor nit: these copyright headers don't have the right comment syntax. I can fix in a follow-up.